eng/Versions.props

```diff
@@ -270,6 +270,7 @@
     <PrivateVisualStudioPackage Include="Microsoft.VisualStudio.TextManager.Interop.10.0"/>
     <PrivateVisualStudioPackage Include="Microsoft.VisualStudio.TextManager.Interop.12.0"/>
     <PrivateVisualStudioPackage Include="Microsoft.VisualStudio.TextManager.Interop.12.1.DesignTime"/>
+    <PrivateVisualStudioPackage Include="Microsoft.VisualStudio.TextManager.Interop.16.0.DesignTime"/>
```

Comment:
Why are we adding this but not using it anywhere?

Reply:
The package was already a dependency but not marked private which caused installing the nuget to fail